deploy/helm/block-explorer/templates/configmap.yaml

```diff
@@ -7,75 +7,360 @@ metadata:
-            "bech32PrefixConsAddr": "sifvalcons",
-            "bech32PrefixConsPub": "sifvalconspub",
-            "bondDenom": "rowan",
-            "powerReduction": 1000000,
-            "coins": [
-                {
-                    "denom": "rowan",
-                    "displayName": "ROWAN",
-                    "displayNamePlural": "ROWAN",
-                    "fraction": 1000000000000000000
-                },
-                {
-                    "denom": "cusdt",
-                    "displayName": "CUSDT",
-                    "displayNamePlural": "CUSDTS",
-                    "fraction": 1000000000000000000
-                },
-                {
-                    "denom": "cusdc",
-                    "displayName": "CUSDC",
-                    "displayNamePlural": "CUSDC",
-                    "fraction": 1000000000000000000
-                },
-                {
-                    "denom": "clink",
-                    "displayName": "CLINK",
-                    "displayNamePlural": "CLINKS",
-                    "fraction": 1000000000000000000
-                },
-                {
-                    "denom": "chot",
-                    "displayName": "CHOT",
-                    "displayNamePlural": "CHOTS",
-                    "fraction": 1000000000000000000
-                }
-            ],
-            "gasPrice": 0.02,
-            "coingeckoId": "sif"
-        },
-        "genesisFile": "{{ .Values.blockExplorer.env.genesisURL }}",
-        "remote":{
-            "rpc": "{{ .Values.blockExplorer.env.remote.rpcURL }}",
-            "lcd": "{{ .Values.blockExplorer.env.remote.lcdURL }}"
-        },
-        "debug": {
-            "startTimer": true,
-            "readGenesis": true
-        },
-        "params":{
-            "startHeight": 0,
-            "defaultBlockTime": 5000,
-            "blockInterval": 15000,
-            "consensusInterval": 1000,
-            "statusInterval":7500,
-            "signingInfoInterval": 1800000,
-            "proposalInterval": 5000,
-            "missedBlocksInterval": 60000,
-            "delegationInterval": 900000
-        }
+       "public":{
+          "chainName":"Sifchain",
+          "chainId":"{{ .Values.blockExplorer.env.chainnet }}",
+          "gtm":"",
+          "slashingWindow":10000,
+          "uptimeWindow":250,
+          "initialPageSize":30,
+          "secp256k1":false,
+          "bech32PrefixAccAddr":"sif",
+          "bech32PrefixAccPub":"sifpub",
+          "bech32PrefixValAddr":"sifvaloper",
+          "bech32PrefixValPub":"sifvaloperpub",
+          "bech32PrefixConsAddr":"sifvalcons",
+          "bech32PrefixConsPub":"sifvalconspub",
+          "bondDenom":"rowan",
+          "powerReduction":1000000,
+          "coins":[
+             {
+                "symbol":"rowan",
+                "decimals":18,
+                "name":"Rowan",
+                "imageUrl":"./images/siflogo.png",
+                "network":"sifchain"
```

Comment:
Unless I'm missing something, this isn't right. See: https://github.com/Sifchain/sifnode/pull/969/files#diff-7033ea6634ac68835b560460888894b0f7ea755c6952be25104b98c44050c082

Reply:
You're right. I just pulled in the frontend config, but this won't parse at all. This will completely break things. I'll close this PR and will raise a new one.